src/main/java/io/debezium/connector/vitess/Vgtid.java

```diff
@@ -94,7 +106,7 @@ public int hashCode() {
         return Objects.hash(rawVgtid, shardGtids);
     }
 
-    public static class ShardGtid {
+    public static class ShardGtid implements Comparable<ShardGtid> {
```

Comment:
JsonPropertyOrder is Jackson I think?

Reply:
Yes, but it's a great point: we should use Jackson really, instead of Gson. Tbh. I hadn't noticed before that Gson would be used here at all, and it's the only connector which does use it. So for the sake of consistency, we should use Jackson indeed, also removing the Gson entry from the Debezium BOM then. Luckily, the Gson usage here is very contained, so this should be a quick change.

Reply:
Done.

Reply:
Perfect, thx!